src/components/SignupForm/SignupForm.js

```diff
@@ -39,22 +39,35 @@ class SignupForm extends Component {
             alert("Verification Email Sent. Please Verify your Email");
           });
         const uid = response.user.uid;
-        console.log(uid);
         const data = {
           id: uid,
           email,
           fullName,
           isImagePresent: false,
           postNumber: 0,
+          adminRights: false,
         };
         const userRef = firebase.firestore().collection("users");
+        const userRef1 = firebase.firestore().collection("userInfo");
```

Comment:
shall we rename the variable name something more convenient and meaningful?

Reply:
done